.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: Node CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```

Comment:
Since this is just a matrix of 1x1, I think you can skip the `strategy` section and replace `${{ matrix.node-version }}` with `12.x`.

Buuuuuuuuuut I also don't mind if we just keep it as is. Then it's easy to add other node versions into the matrix. I'd also note that our other repos that use GitHub CI are testing on 3 versions:

https://github.com/github/auto-complete-element/blob/8c571914da20527facfc8125828c9e01600a9657/.github/workflows/nodejs.yml#L14

Reply:
Actually it seems a little pointless to test across a matrix build because the test command only boots up Karma which then boots up chromeheadless. It almost doesn't matter what Node version we test with.

We should probably spend more time looking into how to test Firefox headless and possibly Edge also.